implementations/rust/ockam/ockam_app/src/invitations/tray_menu.rs

```diff
@@ -100,12 +86,13 @@ fn pending_invitation_menu<R: Runtime>(
             &MenuItemBuilder::with_id(id.clone(), &invitation.recipient_email)
                 .enabled(false)
                 .build(app_handle),
-            &MenuItemBuilder::with_id(
-                format!("invitation-sent-cancel-{}", invitation.id),
-                "Cancel",
-            )
-            .enabled(false)
-            .build(app_handle),
+            // TODO: not supported yet
```

Comment:
I would just remove commented out code. By the time we implement it the variables, data types might have changed and it won't be hard to re-create.

Reply:
I've removed the TODO's blocks 👍